docs/source/release/v3.11.0/framework.rst

```diff
@@ -15,6 +15,7 @@ New
 Improved
 ########
 
+- :ref:`GroupDetectors <algm-GroupDetectors-v2>` now supports workspaces with time indexes.
```

Comment:
Maybe the release notes should refer to scanning/moving detectors, since "time indices" is not meaningful for users (or even other developers).